config/manager/manager.env

```diff
@@ -4,4 +4,4 @@ CSI_REGISTRAR_IMAGE=k8s.gcr.io/sig-storage/csi-node-driver-registrar:v2.3.0
 CSI_PROVISIONER_IMAGE=k8s.gcr.io/sig-storage/csi-provisioner:v3.0.0
 CSI_LIVENESSPROBE_IMAGE=k8s.gcr.io/sig-storage/livenessprobe:v2.5.0
 CSI_RESIZER_IMAGE=k8s.gcr.io/sig-storage/csi-resizer:v1.3.0
-VGMANAGER_IMAGE=quay.io/ocs-dev/vgmanager:latest
+VGMANAGER_IMAGE=quay.io/ocs-dev/lvm-operator:latest
```

Comment:
do we need `VGMANAGER_IMAGE` env if we are fetching the image from the container using `getRunningPodImage`?

Reply:
Yes, incase if you want to custom build only vgmanager and use that image.

Reply:
custom build of vgmanager would mean building the operator image again (since vgmanager is part of operator build)

Reply:
So if the primary use case is to be able to test custom vgmanager builds, then having separate images for both operator and vgmanager would be an ideal option, IMO.

Reply:
- yes, you'll be commenting out vgmanager build and remove env in manager yaml. I can remove this but would like to have this atleast for a week or so and if we aren't seeing it's usage, I'll remove it.
- sounds good or want me to remove it now itself?